412.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {number} n
+ * @return {string[]}
+ */
+var fizzBuzz = function(n) {
+    let answer = [];
+    for (let i = 1; i <= n; i++) {
+        if (i % 15　== 0) {
+            answer[i] = "FizzBuzz";
```

Comment:
It would be better to use `Array.push()` when you add an element to the end of an array.
The way of adding elements above could cause create undefined "holes" in an array.

i.e.)
`[1, 2, undefined, undefined, 5]`

Check out `Array.push()` in [Standard built-in objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects)